module/documents/item.mjs

```diff
@@ -33,6 +33,19 @@ export default class Item5e extends SystemDocumentMixin(Item) {
     return this.system.isActive ?? false;
   }
 
+  /* --------------------------------------------- */
+
+  /**
+   * Does this item require concentration?
+   * @type {boolean}
+   */
+  get isConcentration() {
```

Comment:
`requiresConcentration` might be a better name.

Reply:
And maybe shift it down below `get isVersatile()` so things are bit closer to alphabetical order.